ament_cpplint/test/test_pep257.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ament_pep257.main import main
+
+
+def test_pep257():
+    excluded = ['ament_cpplint/cpplint.py']
```

Comment:
Just wondering, why does this test exclude `ament_cpplint/cpplint.py` while the others just exclude `cpplint.py`? Are they all supposed to be the same path?